core/tests/performance_tests/base.py

```diff
@@ -92,8 +104,23 @@ def _set_test_limits(self, page_config):
         self.load_time_limit_cached_ms = (
             page_config['load_time_limits_secs']['cached'] * 1000)
 
+        if page_config['preload_options']['do_login']:
```

Comment:
Shouldn't page_config be modified accordingly too, so that it contains:

{
    'preload_options': some string
}

?